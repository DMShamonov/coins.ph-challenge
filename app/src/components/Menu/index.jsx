// @flow

import * as React from 'react';
import { NavLink } from 'react-router-dom';

import Routes from 'constants/routes';
import Icon from 'components/Icon';

export default function Menu() {
  return (
    <nav>
      <ul className="menu">
        <li>
          <NavLink exact to={Routes.WALLETS} className="menu__link" activeClassName="menu__link_active">
            <Icon theme="btc-wallet" />
          </NavLink>
        </li>
      </ul>
    </nav>
  );
}
