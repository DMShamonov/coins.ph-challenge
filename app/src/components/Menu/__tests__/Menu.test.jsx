import React from 'react';
import { shallow } from 'enzyme';

import Menu from '../index';

describe('`Menu` component should render correctly', () => {
  test('with default props', () => {
    expect(shallow(<Menu />)).toMatchSnapshot();
  });
});
