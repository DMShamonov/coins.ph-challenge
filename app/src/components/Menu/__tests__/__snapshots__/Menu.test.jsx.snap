// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`\`Menu\` component should render correctly with default props 1`] = `
<nav>
  <ul
    className="menu"
  >
    <li>
      <NavLink
        activeClassName="menu__link_active"
        className="menu__link"
        to="/wallets"
      >
        <Icon
          theme="btc-wallet"
        />
      </NavLink>
    </li>
  </ul>
</nav>
`;
